### PR TITLE
homebrew_cask: Add support for URL bases casks

### PR DESCRIPTION
**Pull request by kyleabenson**
…vely handled by 'brew cask' install/uninstall

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

homebrew_cask.py

##### ANSIBLE VERSION

```
2.3.0
```

##### SUMMARY

Currently homebrew_cask.py does not support URLs for cask installation 

Natively, the brew cask install/uninstall commands support both standard names, local ruby cask files, and url ruby cask files (such as a raw .rb file from a specific commit)

I've added the ability to parse URLs so that they can be installed in the same way as:
brew cask install <URL>

```
ansible localhost -m homebrew_cask -a 'cask=vagrant'
ansible localhost -m homebrew_cask -a 'cask=https://raw.githubusercontent.com/caskroom/homebrew-cask/8bd4de936e673cd895c106fce2a9d740007751fa/Casks/vagrant.rb'
```

### CODE REVIEW

File: lib/ansible/modules/packaging/os/homebrew_cask.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 ---
 module: homebrew_cask
 author:
+    - "Kyle Benson @kyle-benson"
```
Comment (dagwieers) [inline lib/ansible/modules/packaging/os/homebrew_cask.py:28]:
```suggestion
    - Kyle Benson (@kyle-benson)
```

**Review by dagwieers — CHANGES_REQUESTED** [review]
LGTM, needs to have tests in order to be approved.

**Comment by xserrat** [comment]
Hi @dagwieers !

Any updates about this PR?

This PR is just what I need because currently the `virtualbox 6.0.8` is failing and it's the latest version so I would like to force a previous version to avoid this failure...

Thanks so much for you contribution! 👏

**Comment by gundalow** [comment]
@kyleabenson 
Thank you for your PR.
As part of reviewing the backlog of PRs we are looking at PRs older PRs that haven't been updated in a while

Given that:
* PR hasn't been updated since early 2017
* Needed a rebase since the beginning of the year
* This is a feature rather than a bug fix

Therefore I'm going to close this.

If you or anyone else wants to continue with this work then please do feel free to create a fresh PR and `@mention` the previous reviewers in here.

**Pull request closed without merging**